### ISSUE TITLE
libkdumpfile doesnt work with drgn

### ISSUE THREAD

**Issue by bgly** [osandov/drgn#63]
I am using gcc version 7.3.1 and whenever I build libkdumpfile with the latest drgn it is never detected.

Currently using centos 7 with 4.19 kernel

```
[root@bl-vsnap-100 drgn]# drgn -c /var/crash/127.0.0.1-2020-06-18-17\:18\:49/vmcore
Traceback (most recent call last):
  File "/usr/local/bin/drgn", line 11, in <module>
    load_entry_point('drgn==0.0.5', 'console_scripts', 'drgn')()
  File "/usr/local/lib64/python3.6/site-packages/drgn-0.0.5-py3.6-linux-x86_64.egg/drgn/internal/cli.py", line 113, in main
    prog.set_core_dump(args.core)
ValueError: drgn was built without libkdumpfile support

[root@bl-vsnap-100 ~]# sdb /usr/lib/debug/lib/modules/4.19.119-2c.el7.x86_64/vmlinux /var/crash/127.0.0.1-2020-06-18-17\:18\:49/vmcore
Traceback (most recent call last):
  File "/usr/local/bin/sdb", line 11, in <module>
    load_entry_point('sdb==0.1.0', 'console_scripts', 'sdb')()
  File "/usr/local/lib/python3.6/site-packages/sdb-0.1.0-py3.6.egg/sdb/internal/cli.py", line 213, in main
    prog = setup_target(args)
  File "/usr/local/lib/python3.6/site-packages/sdb-0.1.0-py3.6.egg/sdb/internal/cli.py", line 158, in setup_target
    prog.set_core_dump(args.core)
ValueError: drgn was built without libkdumpfile support
```

**Comment by sdimitro** [comment]
Hi @bgly ! In the sdb repo we have daily automation that builds the latest libkdumpfile and then the latest drgn so it can test sdb with bleeding-edge (here is a run from 17hrs ago -> https://github.com/delphix/sdb/runs/782551066?check_suite_focus=true).

If I were to guess, I'd say that this is an environment issue in the machine that you are trying this out (e.g. maybe the libkdumpfile is installed in a different directory than the place that drgn expects it? etc..). Can you provide more details on how you've installed things and where to the artifacts of the installation end up?

**Comment by sdimitro** [comment]
BTW this is the `drgn` repo so it would be more helpful for others to have examples running the `drgn` utility instead of `sdb`

**Comment by bgly** [comment]
I pull down the libkdumpfile repo then run:
```
autoreconf -fi
./configure
make
make install
```

Then I pull down drgn and run:
```
python3 setup.py build
python3 setup.py install
```

**Comment by bgly** [comment]
Default pkg-config 0.27 version in centos does not recognize libkdumpfile when installed. The fix was to upgrade pkg-config to 0.29 and rebuild everything
